src/containers/Home.js

```diff
@@ -68,9 +75,8 @@ class Home extends Component {
 
     return (
       <div className="authed">
-        {authorizedRedirect}
         <h1>Welcome to Reggie web</h1>
-        <h2>{this.props.errorMessages}</h2>
+        <h2>{this.props.errorMessages[0]}</h2>
```

Comment:
this seems too explicit. What if there are multiple error messages? It may be best to abstract this into its own component eventually.

Reply:
that's part of the user experience thing. don't want to show the error message from backend, but still want to post it to web so someone would debug....it's like the sso website where it hides the actual err message.